hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestHRegionFileSystem.java

```diff
@@ -380,4 +382,190 @@ public void testTempAndCommit() throws IOException {
 
     fs.delete(rootDir, true);
   }
+
+  @Test
+  public void testSplitStoreDir() throws Exception {
```

Comment:
Better place the below methods to a different test class? And is it possible to only start mini cluster once? Now every test method needs to start and then stop the mini cluster, which is time consuming...

Reply:
Ok, let me move these to a separate class and do the mini cluster init only once.